### PR TITLE
Filelocations

### PR DESCRIPTION
**Pull request by muryoh**

### CODE REVIEW

File: src/main/groovy/com/ullink/NuGetRestore.groovy [hunk 1]
```diff
@@ -19,20 +18,29 @@ class NuGetRestore extends BaseNuGet {
...
-            commandLineArgs += file
-            if (restoreFolder) {
-                commandLineArgs += "-RestoreFolder"
-                commandLineArgs += restoreFolder
+            } else {
+                commandLineArgs += file
             }
         }
+        if (packagesDirectory) {
+            commandLineArgs += "-SolutionDirectory"
```
Comment (gluck) [inline src/main/groovy/com/ullink/NuGetRestore.groovy]: inversed options ?

File: src/main/groovy/com/ullink/NuGetRestore.groovy [hunk 2]
```diff
@@ -19,20 +18,29 @@ class NuGetRestore extends BaseNuGet {
...
-            if (!file.exists())
-            {
+            if (!file.exists()) {
                 logger.info("NuGet repositories.config not found at path: ${file}, running default package restore.");
-                return commandLineArgs;
-            }
-
-            commandLineArgs += file
-            if (restoreFolder) {
-                commandLineArgs += "-RestoreFolder"
```
Comment (gluck) [inline src/main/groovy/com/ullink/NuGetRestore.groovy]: option doesn't exist, is that it ?

File: src/main/groovy/com/ullink/NuGetRestore.groovy (hunk 1 above)
Comment (muryoh) [inline src/main/groovy/com/ullink/NuGetRestore.groovy]: Meh, sorry about that

File: src/main/groovy/com/ullink/NuGetRestore.groovy (hunk 2 above)
Comment (muryoh) [inline src/main/groovy/com/ullink/NuGetRestore.groovy]: Yes it's the one that doesn't exist. Don't know why it's here :P

File: src/main/groovy/com/ullink/NuGetRestore.groovy [hunk 3]
```diff
@@ -19,20 +18,29 @@ class NuGetRestore extends BaseNuGet {
     @Override
     List<String> extraCommands() {
         def commandLineArgs = new ArrayList<String>()
+
         if (projectFolder) {
             def file = new File(projectFolder, "repositories.config");
```
Comment (muryoh) [inline src/main/groovy/com/ullink/NuGetRestore.groovy]: I also don't know what we're doing with this option

Comment (gluck) [inline src/main/groovy/com/ullink/NuGetRestore.groovy]: Maybe it was used/useful in the past, drop it it you'd like

**Pull request merged**